_data/marynaggita/README.md

```diff
@@ -0,0 +1,55 @@
+# [marynaggita](https://github.com/marynaggita/)'s Zarr Collection
+
+## Tweet Mentions
+
+- https://twitter.com/latlong_blog/status/1543726888908009472
+- https://twitter.com/_jhamman/status/1579882167303168004
+- https://twitter.com/ErikvanSebille/status/1578030942144188417
+
+## Zarr Use Cases
+- Day's worth of analysis files for a single variable and combine them using high-level APIs.(https://mesowest.utah.edu/html/hrrr/zarr_documentation/html/xarray_one_day_analysis_example.html)
+
+- Analyzing MOM6 with a python/xarray/dask/zarr software stack (https://www.cesm.ucar.edu/events/2020/MOM6/files/2020-MOM6-Dussin.pdf)
+
+- Test ZARR performance for CAB-LAB data cubes (https://notebook.community/CAB-LAB/cablab-cubeio/notebooks/zarr_cube)
+
+- Beyond netCDF: Cloud Native Climate Data with Zarr and XArray(https://ui.adsabs.harvard.edu/abs/2018AGUFMIN33A..06A/abstract)
+
+## Zarr Data Visualisation
+- Boundaries of active fires (red outlines) estimated using VIIRS 375-m thermal anomalies, and smoke from wildfires in the Pacific Northwest on 9 Sep 2020 (https://worldview.earthdata.nasa.gov/)
+
+<img src="https://github.com/marynaggita/beautiful-zarr/blob/marynaggita/_data/marynaggita/Screenshots/data4.jpg" width="400" height="400" display="block" margin= "0 auto">
```

Comment:
This visualisation looks excellent, but the data visualised here is not in Zarr format. It's in `HDF-EOS` format Can you please remove it or add another Zarr data visualisation? Thanks!

Reply:
I have removed it. Thanks so much

Reply:
Thank you.